### PR TITLE
fix(help-menu): Only show email if user is paying

### PR DESCRIPTION
**Pull request by timgl**
## Problem

We don't have a great process for giving free users support over email. Instead, encourage them to ~join Slack~ create an issue or ask our docs.

## Changes

Only show email if the user has an active subscription.

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/components/HelpButton/HelpButton.tsx [hunk 1]
```diff
@@ -135,18 +122,33 @@ export function HelpButton({
                         >
                             Create an issue on GitHub
                         </LemonButton>
+                        {billing?.current_total_amount_usd > 0 && (
```
Comment (Twixes) [inline frontend/src/lib/components/HelpButton/HelpButton.tsx:125]: ⚠️ There's a TypeScript error here.
BTW why check spend and not the plan? Spend may be zero at the beginning of a billing period, right?

**Review by Twixes — CHANGES_REQUESTED** [review]
Makes sense, though there's an error from TypeScript and a question from me

File: frontend/src/lib/components/HelpButton/HelpButton.tsx (hunk 1 above)
Comment (timgl) [reply frontend/src/lib/components/HelpButton/HelpButton.tsx]: Fair point. I've changed it to be the total predicted spend, and set a minimum of 1.6k/month spend.

File: frontend/src/scenes/billing/billingLogic.ts [hunk 2]
```diff
@@ -104,6 +104,20 @@ export const billingLogic = kea<billingLogicType>([
             (billing, billingLoading): BillingVersion | undefined =>
                 !billingLoading || billing ? (billing ? 'v2' : 'v1') : undefined,
         ],
+        totalPredictedSpend: [
```
[the remark this answers is not in the document]
Comment (Twixes) [reply frontend/src/scenes/billing/billingLogic.ts:107]: Some `convertUsageToAmount` Jest tests need to be update now.
As for testing this, I'd suggest adding two test cases as `billingLogic.test.ts`, though not a blocker

**Review by kappa90 — APPROVED** [review]
There is already a `projected_amount_usd` in the API, but it wasn't exposed in the Typescript interface. So we can just iterate on that and sum it across products.

**Comment by timgl** [comment]
Just chatted with Simon, we'll actually allow this for everyone that's put their card in so will change that.

File: frontend/src/lib/lemon-ui/icons/icons.tsx [hunk 3]
```diff
@@ -1732,19 +1732,19 @@ export function IconSlack(props: LemonIconProps): JSX.Element {
             <g clipRule="evenodd" fillRule="evenodd">
                 <path
                     d="m897.4 0c-135.3.1-244.8 109.9-244.7 245.2-.1 135.3 109.5 245.1 244.8 245.2h244.8v-245.1c.1-135.3-109.5-245.1-244.9-245.3.1 0 .1 0 0 0m0 654h-652.6c-135.3.1-244.9 109.9-244.8 245.2-.2 135.3 109.4 245.1 244.7 245.3h652.7c135.3-.1 244.9-109.9 244.8-245.2.1-135.4-109.5-245.2-244.8-245.3z"
-                    fill="#36c5f0"
+                    fill={props.color || '#36c5f0'}
```
Comment (Twixes) [inline frontend/src/lib/lemon-ui/icons/icons.tsx:1735]: This icon doesn't seem to actually be used, so this doesn't do anything?

**Review by Twixes — APPROVED** [review]
This makes sense!

**Comment by Twixes** [comment]
You will need to update those Jest unit tests though.

**Comment by tiina303** [comment]
I assume we can close this as we went a different route with support

**Pull request closed without merging**